crypto/src/key/rschnorr/mod.rs

```diff
@@ -16,14 +16,12 @@
 mod internal;
 
 use crate::random::{CryptoRng, Rng};
-use generic_array::GenericArray;
 pub use internal::add_sigs;
 pub use internal::RistrettoSchnorrSignature;
-use internal::*;
 use serialization::{Decode, Encode};
-use tari_crypto::{keys::PublicKey, tari_utilities::ByteArray};
+use zeroize::Zeroize;
```

Comment:
If we're doing this manually we should probably consider doing a "proper job" here. Not for this PR though. Might be worth a todo for the future though